nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-actions.js

```diff
@@ -1271,7 +1272,43 @@ nf.Actions = (function () {
                                 // clear the template dialog fields
                                 $('#new-template-name').val('');
                                 $('#new-template-description').val('');
-                            }).fail(nf.Common.handleAjaxError);
+                            }).fail(function () {
```

Comment:
Here, I'm saying that I should propose users to override an existing template in case of failure during the first `createTemplate` attempt. However the failure could be caused by something else than "template already exists". I'd like some "UI guru" advice regarding the best way to only catch the wanted error and to execute `nf.Common.handleAjaxError` otherwise.